pkg/meta/annotations.go

```diff
@@ -197,6 +197,15 @@ const (
 	NMAHealthProbeInitialDelaySeconds = "initial-delay-seconds"
 	//
 	// Use GenNMAHealthProbeAnnotationName to generate the name.
+
+	// Set of annotations for scrutinize
+	//
+	ScrutinizePodTimeToLive             = "vertica.com/scrutinize-pod-ttl"
```

Comment:
Suggest we create comments for each annotation. Also, we usually include the term `Annotation` in the variable.
```suggestion
        // Controls how long the scrutinize pod will keep running. The time is specified in seconds.
	ScrutinizePodTTLAnnotation             = "vertica.com/scrutinize-pod-ttl"
```